okhttp-tests/src/test/java/okhttp3/URLConnectionTest.java

```diff
@@ -104,10 +104,13 @@
 import static okhttp3.mockwebserver.SocketPolicy.SHUTDOWN_OUTPUT_AT_END;
 import static okhttp3.mockwebserver.SocketPolicy.UPGRADE_TO_SSL_AT_END;
 import static okhttp3.tls.internal.TlsUtil.localhost;
+import static org.hamcrest.CoreMatchers.either;
```

Comment:
I'd like to migrate to AssertJ or Truth. We've resisted it to make ingesting OkHttp easier for AOSP; I think it's probably a bad trade off.